sauce/features/accounts/auto-distribute-splits/index.js

```diff
@@ -24,17 +22,24 @@ export class AutoDistributeSplits extends Feature {
   }
 
   observe(changedNodes) {
-    if (changedNodes.has(SPLIT_BUTTON_CLASS) && !this.buttonPresent()) {
-      this.addButton();
+    if (!changedNodes.has('ynab-grid-actions-buttons')) {
+      if (
+        $('.ynab-grid-cell-subCategoryName input').val() ===
+        'Split (Multiple Categories)...'
```

Comment:
This would have problems when mixed with the internationalisation feature. Would be best to check something else about the button rather than the description.